homeassistant/components/melnor/sensor.py

```diff
@@ -116,7 +114,7 @@ def __init__(
         self.entity_description = entity_description
 
         self._attr_unique_id = (
-            f"sensor-{self._attr_unique_id}-{self.entity_description.key}"
+            f"sensor-{super().unique_id}-{self.entity_description.key}"
         )
```

Comment:
We should not both set `_attr_unique_id` and implement a `unique_id` property, do only one.

Reply:
Yeah this was an oops. Caught it while self-reviewing. Already pushed a fix.